docs/detections/alerts-reduce.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[reduce-notifications-alerts]]
+== Reduce notifications and alerts
+
+{elastic-sec} offers several features to help reduce the amount of notifications and alerts produced by your detection rules. This table provides a general comparison of these features, with links for more details:
+
+[cols="2"]
+|===
+
+| <<snooze-rule-actions,Rule action snoozing>>
+a| *_Prevents a specific rule's notification actions from running_*. 
+
+Use to avoid unnecessary notifications from a noisy rule. The rule continues to run and generate alerts during the snooze period, but its <<rule-notifications,notification actions>> are suppressed.
```

Comment:
While this can be used to quiet a noisy rule, it can also be used to temporarily stop notifications from being sent during certain timeframes (e.g., during application/network maintaince).

```suggestion
Use to avoid unnecessary notifications from specific rules. The rule continues to run and generate alerts during the snooze period, but its <<rule-notifications,notification actions>> are suppressed.
```

Reply:
Good call, let's actually include all those as examples:
```suggestion
Use to avoid unnecessary notifications from a specific rule. For example, you can turn off a rule's notifications during application or network maintenance. The rule continues to run and generate alerts during the snooze period, but its <<rule-notifications,notification actions>> are suppressed.
```

Reply:
```suggestion
Use to avoid unnecessary notifications from a specific rule. The rule continues to run and generate alerts during the snooze period, but its <<rule-notifications,notification actions>> are suppressed.
```